docs/Golden-Gate/Output-Files.rst

```diff
@@ -0,0 +1,30 @@
+Output Files
+============
+
+Golden Gate generates many output files, we describe them here.  Note, the GG
+CML-argument ``--base-output-filename=<BASE>`` defines defines a common prefix
```

Comment:
Perhaps it's best to just do what RC did?
https://github.com/chipsalliance/rocket-chip/blob/master/src/main/scala/stage/RocketChipAnnotations.scala#L42

I like prefix personally, but base i used throughout FIRRTL.

Reply:
Ugh, `--name` and `-n`?  They should probably have just used `--option` and `-o`.  Description much?  I think base or prefix is fine.  Including `output-filename` in the long-name argument is great because it describes what the base or prefix or name is dealing with.